mm2src/mm2_main/src/iris_swap_poc.rs

```diff
@@ -0,0 +1,263 @@
+use super::*;
+use mm2_test_helpers::for_tests::{enable_eth_coin, enable_tendermint, iris_nimda_testnet_conf, iris_testnet_conf,
+                                  tbnb_conf, usdc_ibc_iris_testnet_conf, RICK_ELECTRUM_ADDRS};
+
+// https://academy.binance.com/en/articles/connecting-metamask-to-binance-smart-chain
+const TBNB_URLS: &[&str] = &["https://data-seed-prebsc-1-s1.binance.org:8545/"];
+// https://testnet.bscscan.com/address/0xb1ad803ea4f57401639c123000c75f5b66e4d123
+const TBNB_SWAP_CONTRACT: &str = "0xB1Ad803ea4F57401639c123000C75F5B66E4D123";
+
+#[test]
+#[ignore]
+// cargo test mm2::mm2_tests::iris_swap_poc::test -- --exact --ignored
+fn test() {
+    let pairs = [
+        ("USDC-IBC-IRIS", "IRIS-NIMDA"),
+        ("IRIS-NIMDA", "RICK"),
+        ("USDC-IBC-IRIS", "tBNB"),
+    ];
+    block_on(trade_base_rel_iris(&pairs, 1, 2, 0.01));
```

Comment:
Can we trade at least one pair in an unignored test?

Reply:
Seeds are hard-coded for now, so it may cause problems if multiple CI agents run tests simultaneously. We will need to use passphrases from environment variables and fund corresponding addresses with testnet coins. I think it's feasible to do it this week, will cover.

Reply:
@sergeyboyko0791 I decided to do it on the next iteration, added to the checklist: https://github.com/KomodoPlatform/atomicDEX-API/issues/1432#issuecomment-1250850653